### DISCUSSION TITLE
Release v1.14.7 / 5.69.7

### DISCUSSION THREAD

**Discussion by DavidXanatos** [sandboxie-plus/Sandboxie#4208] [Announcements]
In this release, we have introduced several new features and improvements that significantly enhance the user experience and provide greater flexibility in system operations. Most notably, users can now effortlessly obtain free 10-day evaluation certificates directly from the support settings page within the UI. These certificates are hardware-locked to the user's machine and allow for up to three requests per hardware ID, making it easier to test and evaluate the system with minimal setup.

Furthermore, new options have been added to increase privacy and security, such as the ability to modify the Windows Product ID in the registry to a random value and to return random values for disk serial numbers and network adapter MAC addresses when queried by applications. These features add an extra layer of obfuscation to protect against unwanted system identification.

Other enhancements include the ability to terminate all processes when Sandman exits, a new option for configuring DropConHostIntegrity directly from the UI, and an improved shared template feature in the New Box Wizard. The number of available shared templates has increased to 10, and the template names can now be easily updated by adjusting the corresponding settings.

In terms of fixes, we have addressed several key issues, including improving the "HideDiskSerialNumber" functionality to prevent application crashes, correcting the format of encrypted proxy passwords, and resolving an issue related to the "NtQueryDirectoryObject" function to avoid easy sandbox detection. These updates contribute to a more stable and secure environment for users.

For a full list of changes and fixes please review the [full change log](https://github.com/sandboxie-plus/Sandboxie/blob/master/CHANGELOG.md#1147--5697---2024-09-05).

#### You can support the project through donations, any help will be greatly appreciated.

[<img src="https://www.paypalobjects.com/en_US/AT/i/btn/btn_donateCC_LG.gif">](https://xanasoft.com/go.php?to=donate)

#### If you have issues with an update installation, just uninstall the previous version keeping the sandboxie.ini and reinstall the new build.

<hr /><em>This discussion was created from the release <a href='https://github.com/sandboxie-plus/Sandboxie/releases/tag/v1.14.7'>Release v1.14.7 / 5.69.7</a>.</em>

**Comment by bastik-1001** [comment]
The file: `Sandboxie-Plus-x64-v1.14.7.exe` is signed and has the following hashes:

```
CRC32: E00982A1
CRC64: BC49104B61051DC9
MD5: AA39363E6EDB03185DC2E108CFA7938F
SHA-1: 5E6B0ACDC95D7AAF225210CE2DE163AF12D6D1AA
SHA-256: 7EA4FB1C3B4A828126D19A75141553DB003D6D2E0B4F4D2225F1AA71EDB9C840
SHA-512: FF96EB71AA385B7D234E2D27EF7E182F01BAC0D9277B5570861C1CEFB13AF33F2D5BBE3C73C82FECA8B7BEED359CDED4257C2CF8109096437FBE5B43777D2E30

```
